### ISSUE TITLE
R1Q9 Large cell

### ISSUE THREAD

**Issue by dagewa** [dagewa/DIALS_for_ED_paper#21]
> The unit cell is rather large. How many pixels were between the reflections along the c* axis? How is this length related to the geometrical scattering error in the data? How good were the spots separated along the c* at all – I assume, the crystals were laying with the c* parallel to the incident beam, this is a problematic direction in general. Any diffuse scattering in the data?

**Comment by dagewa** [comment]
@jmp1985 how easy is it to determine how well-separated spots are along one direction (_c_*) from an `integrated_experiments.json` and `integrated.pickle`?

**Comment by dagewa** [comment]
perhaps it suffices merely to report how many overlaps there are? If this number is near zero then there should not be a problem

**Comment by dagewa** [comment]
So c* is closest to the beam for dataset 6, which is also the worst dataset. The beam is close to being in the plane c*-b* for this dataset. There may well be some streaking along this direction, but it is hard to tell from the images alone.

**Comment by cmaxtb** [comment]
We didn't really make any attempt to quantify diffuse scattering, of course diffuse scattering events will be present in our data and contribute to the background, but the background is rather low overall, there are images of a spot profile in the lysozyme paper showing the immediate background surrounding the peak, we might refer to that 

The carbon support on the grid will also scatter, when you merge frames you start to see the diffuse carbon rings

We did not use an energy filter, this would have removed most of the inelastic scattering

**Comment by dagewa** [comment]
Drafting a reply:
Considering the cell used for integration (i.e. a~32, b~68, c~105), the crystals were not all lying with c* parallel to the incident beam. Preferred orientation can be investigated within the dials.reciprocal_lattice_viewer. This indicates a tendency for a* to lie orthogonal to the beam, but no clearly strong preference for the alignment of the other axes. The dataset with the smallest angle between c* and the beam direction was the most problematic dataset, number 6, however even there the angle was greater than 40 degrees. Spot separation was sufficient for successful integration of all datasets. We did not see any evidence of diffuse scatter, but we did not specifically look for it.

**Comment by dagewa** [comment]
@jmp1985 we might be able to produce numbers for the fraction of reflections that have peak regions overlapping with other reflections?

**Comment by dagewa** [comment]
Reply:
Considering the cell used for integration (i.e. a=32, b=68, c=105) rather than that re-indexed for merging in P21212, the crystals were *not* generally lying with c* parallel to the incident beam. We investigated preferred orientation using the dials.reciprocal_lattice_viewer. This indicates a tendency for a* to lie orthogonal to the beam, but no clearly strong preference for the alignment of the other axes. The dataset with the smallest angle between c* and the beam direction was the most problematic dataset, number 6, however even there the angle was greater than 40 degrees. Visual inspection of images with dials.image_viewer indicates that overlaps were not a significant issue for these datasets. Currently DIALS does not have tools for easily determining the degree of overlap between reflections during integration, but these features are in development. We did not see any evidence of diffuse scatter, but we did not specifically look for it. Background counts are in general low for all datasets (see for example Figure 2 in Clabbers _et al._ (2017)).
